pkg/codegen/python/gen_test.go

```diff
@@ -57,7 +59,26 @@ func TestRelPathToRelImport(t *testing.T) {
 }
 
 func TestGeneratePackage(t *testing.T) {
-	test.TestSDKCodegen(t, "python", GeneratePackage)
+	test.TestSDKCodegen(t, "python", GeneratePackage, typeCheckGeneratedPackage)
+}
+
+// We can't type check a python program. We just check for syntax errors.
```

Comment:
We kind of can with mypy but it's not been working very well. I think this is a fine choice here 👍